src/ThemeProvider/index.tsx

```diff
@@ -20,25 +20,13 @@
 import { Interpolation, ThemeContext, ThemeProvider as EmotionThemeProvider } from '@emotion/react';
 import { CreateStyled, default as emotionStyled } from '@emotion/styled';
 import * as React from 'react';
-
 import defaultTheme from '../theme/defaultTheme';
 
-const themes = {
-  default: defaultTheme,
-};
-
-const ThemeProvider: React.ComponentType<
-  React.PropsWithChildren<{ theme?: keyof typeof themes }>
-> = ({ theme = 'default', children }) => {
-  return (
-    <EmotionThemeProvider theme={themes[theme]}>
-      <link
-        href={'https://fonts.googleapis.com/css?family=Work+Sans:300,400,600&display=swap'}
-        rel="stylesheet"
-      />
-      {children}
-    </EmotionThemeProvider>
-  );
+const ThemeProvider: React.ComponentType<React.PropsWithChildren<{ theme?: any }>> = ({
```

Comment:
Theme: any?

Reply:
Discussed in Slack, this would be a good place for a comment since we are using `any` intentionally